util/util.go

```diff
@@ -254,3 +255,19 @@ func IsRunningSystemd() bool {
 	}
 	return fi.IsDir()
 }
+
+// GetMachineID returns a host's 128-bit machine ID as a string.
+func GetMachineID() (machineID string, err error) {
```

Comment:
A few style nits for this function:
a) I'd omit the named result parameters - they add little value and it stutters a bit in godoc
b) Error strings should start [lower-case](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
b) I would rather promote the comment to the docstring, similar to that of `IsRunningSystemd`, e.g.: "This functions similar to systemd's `sd_id28_get_machine`, ..." 
c) Let's just `strings.TrimSpace` instead of manual string slicing